infra/resources/_modules/function_apps/locals.tf

```diff
@@ -65,7 +65,11 @@ locals {
       WEBSITE_SWAP_WARMUP_PING_PATH     = "/api/v1/wallet/health"
       WEBSITE_SWAP_WARMUP_PING_STATUSES = "200"
       },
-      local.function_apps.common_app_settings
+      local.function_apps.common_app_settings,
+      {
+        for s in var.support_func.app_settings :
+        s.name => s.key_vault_secret_name != null ? "@Microsoft.KeyVault(VaultName=${var.project}-wallet-kv-01;SecretName=${s.key_vault_secret_name})" : s.value
+      }
```

Comment:
Why is this hardcoded?

Reply:
changed in f6dc59526dcd5a8b72dee0e5219ee081873f1401